.github/workflows/build.yaml

```diff
@@ -60,7 +60,6 @@ jobs:
       - uses: taiki-e/install-action@v2
         with:
           tool: just@1
+      - name: Setup publication key
```

Comment:
should we be paranoically causious and remove the file explicitly after just publish? In case the VM get's stuck, we don't want this to lie around, maybe.

Reply:
Yes, we can do that. I just pushed a change to always remove that file at the end of the job